### PR TITLE
Change the download task to make three attempts before failing.

### PR DESCRIPTION
**Pull request by JeremyKuhne**
Transient web issues are causing build failures. @weshaggard

### CODE REVIEW

**Comment by ellismg** [comment]
LGTM

File: dir.targets [hunk 1]
```diff
@@ -17,7 +17,30 @@
...
+
+            while (tryCount <= maxTries)
+            {
+                try
+                {
+                    Log.LogMessage("Attempting to download {0}...", Address);
+                    client.DownloadFile(Address, tempFile);
+                    break;
+                }
+                catch (System.Net.WebException e)
```
Comment (stephentoub) [inline dir.targets]: Do we always want to retry like this, or do e need to distinguish between cases where the server said the file wasn't available vs the server couldn't be reached/etc.?

**Comment by stephentoub** [comment]
One question in a comment, otherwise LGTM.

File: dir.targets (hunk 1 above)
Comment (JeremyKuhne) [inline dir.targets]: I didn't figure it was worth distinguishing as the other sorts of errors shouldn't come up often and will get logged anyway. Leaning towards simpler given that it is an inline task.

**Pull request merged**